packages/ckeditor5-dev-tests/lib/tasks/runmanualtests.js

```diff
@@ -19,19 +19,35 @@ const transformFileOptionToTestGlob = require( '../utils/transformfileoptiontote
  * Main function that runs manual tests.
  *
  * @param {Object} options
- * @param {Array.<String>} options.files
+ * @param {Array.<String>} options.files Glob patterns specifying which tests to run.
+ * @param {String} options.themePath A path to the theme the PostCSS theme-importer plugin is supposed to load.
+ * @param {String} [options.language] A language passed to `CKEditorWebpackPlugin`.
+ * @param {String} [options.additionalLanguages] Additional languages passed to `CKEditorWebpackPlugin`.
```

Comment:
I'd expect `Array.<String>` here. Maybe you can parse it in the `parseArgument` fn?